llama_index/agent/openai_agent.py

```diff
@@ -333,14 +375,14 @@ def chat(
         self,
         message: str,
         chat_history: Optional[List[ChatMessage]] = None,
-        function_call: Union[str, dict] = "auto",
+        tool_choice: Union[str, dict] = "auto",
```

Comment:
this is a user facing breaking change right?

I wonder if we should just bump to 0.9.

Reply:
yeah it is. 

that's a good idea i think